lib/govuk_publishing_components/presenters/cookie_banner_helper.rb

```diff
@@ -9,8 +9,8 @@ def initialize(local_assigns)
         @local_assigns = local_assigns
       end
 
-      def confirmation_message
-        'You\'ve accepted all cookies. You can <a class="govuk-link" href="/help/cookies" data-module="track-click" data-track-category="cookieBanner" data-track-action="Cookie banner settings clicked from confirmation">change your cookie settings</a> at any time.'.html_safe
+      def confirmation_link
+        'You can <a class="govuk-link" href="/help/cookies" data-module="track-click" data-track-category="cookieBanner" data-track-action="Cookie banner settings clicked from confirmation">change your cookie settings</a> at any time.'.html_safe
```

Comment:
This looks non-dynamic. Why not just put this whole bit of markup in the template? Is it used in other places?